src/settings.h

```diff
@@ -157,7 +157,7 @@ class CClientSettings : public CSettings
     int              iNumMixerPanelRows;
     CVector<QString> vstrCentralServerAddress;
     ECSAddType       eCentralServerAddressType;
-    int              iCustomDirectoryIndex;
+    int              iCustomDirectoryIndex;  // index of selected custom central server
```

Comment:
I guess my concern is that this appears in `src/settings.h` but not `src/settings.cpp` -- I'd expect at least to see something initialising it - the constructor for `CClientSettings`, for example, like `eCentralServerAddressType ( AT_DEFAULT ),`.  (I don't like letting the compiler decide on values.  Yes, it'll _probably_ put a zero in... if it bothers to generate code to do that.)

Reply:
Yes you are right.

Reply:
So funny the file is named .ini